packages/vc-verification/src/credential-resolver.ts

```diff
@@ -1,4 +1,5 @@
-import { IVerifiableCredential } from './models';
```

Comment:
If we're removing `IVerifiableCredential` usage from the `ew-credentials` can we remove the actual file? https://github.com/energywebfoundation/ew-credentials/blob/31a697c8f8603b7995794b962cdb1dacccd4bb67/packages/vc-verification/src/models/verifiable-credentials.ts#L1 

It looks like this might be a breaking change (to our exports) though so maybe it would make sense to also export `VerifiableCredential` so that clients don't also have to install `'@ew-did-registry/credentials-interface'`

What are your thoughts here?

Reply:
@jrhender sure, so import @ew-did-registry/credentials-interface and export it from this package? is that right?

Reply:
Yes, that's what makes sense to me

Reply:
Thanks for the further changes @whitneypurdum . I think it still make sense to remove `IVerifiableCredential` as it seems like it is not being used anymore and is effectively replaced by `VerifiableCredential`

Reply:
@jrhender Ah yes, sorry I forgot to do that. Do you think I should check if its being used in SSI hub or elsewhere? I dont think ICL is using it anymore.

Reply:
I would just remove it IMO. A breaking change is fine at this point

Reply:
@jrhender yes it’s already removed

Reply:
@whitneypurdum are the other interfaces like `CredentialSubject`, `Role`, `IssuerFields` and `Proof` still needed or can those be removed too now that `IVerifiableCredential` is removed?